### ISSUE TITLE
Change examples' javadoc to show how to use appassembler

### ISSUE THREAD

**Issue by mziccard** [GoogleCloudPlatform/gcloud-java#863]
Right now our example classes contain the following javadoc (example taken from resourcemanager):

```
Steps needed for running the example:

login using gcloud SDK - gcloud auth login.
compile using maven - mvn compile
run using maven - mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.resourcemanager.ResourceManagerExample" -Dexec.args="[list | [create | delete | get] projectId]"
```

Once `gcloud-java-nio` is merged we will move to `appassembler` hence javadoc should be updated accordingly. /cc @aozarov @jean-philippe-martin

**Comment by jean-philippe-martin** [comment]
Oh you're right! We could fix those in the gcs-nio branch already.

**Comment by mziccard** [comment]
This was fixed in #865
